src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/nocodeinfile/Example2.java

```diff
@@ -6,8 +6,12 @@
 </module>
 */
 
+// violation 8 lines above
+
 // xdoc section -- start
-/* // violation
- block comment
+/*
```

Comment:
Above this line and below `// xdoc section` please add comment `// the violation on first line of file `
 Do same for another example.

Reason: final Rendering of web page looks strange and not clear where is violation

Reply:
> final Rendering of web page looks strange and not clear where is violation

I agree, added the message